src/analysis/seminorm.lean

```diff
@@ -348,6 +389,21 @@ begin
   ...    < r   : by rwa mem_ball_zero at hy,
 end
 
+lemma ball_finset_sup_eq_Inter (p : ι → seminorm 𝕜 E) (s : finset ι) (e : E) (r : ℝ) (hr : 0 < r) :
```

Comment:
maybe make the e, r implicit arguments?

Reply:
`r` yes, `e` that's arguable

Reply:
I agree with `r`, but don't see why `e` would be.